tools/ttysnoop_example.txt

```diff
@@ -73,11 +73,16 @@ positional arguments:
   device         path to a tty device (eg, /dev/tty0) or pts number
 
 optional arguments:
-  -h, --help     show this help message and exit
-  -C, --noclear  don't clear the screen
+  -h, --help      show this help message and exit
+  -C, --noclear   don't clear the screen
+  -s, --datasize  size of the transmitting buffer (default 4096)
+  -c, --datacount number of times we check for 'datasize' data (default 20)
```

Comment:
Please add these new options to man page as well.

Reply:
ok